knowledge-base/loader-container-center-in-viewport.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Center LoaderContainer in viewport
+description: How to center the LoaderContainer in viewport
+type: how-to
+page_title: Center LoaderContainer in viewport
+slug: loader-container-kb-center-in-viewport
+position: 
+tags: loadercontainer, center, viewport, long, page
+ticketid: 1564614
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>LoaderContainer for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+We use the TelerikLoaderContainer component on our pages to let the user know that we're doing something in the background. Some of our pages are twice or three times the height of the user's viewport in their browser. When the user takes an action that could take some length of time we display the TelerikLoaderContainer but it always displays in the center of the page as a whole, not in the center of the browser's viewport. So, if the user is at the top or bottom of the page they won't see the loader animation.
```

Comment:
Generally, there is no need to copy-paste everything the customer said. You can extract all the important keywords and use them in a shorter paragraph.